[2023-11-20T08:38:36.288+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-02T00:00:00+00:00 [queued]>
[2023-11-20T08:38:36.295+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-02T00:00:00+00:00 [queued]>
[2023-11-20T08:38:36.295+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-20T08:38:36.314+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): retrieve_data> on 2023-11-02 00:00:00+00:00
[2023-11-20T08:38:36.328+0000] {standard_task_runner.py:57} INFO - Started process 75 to run task
[2023-11-20T08:38:36.334+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'austin_animal_shelter_dag', 'retrieve_data', 'scheduled__2023-11-02T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmppihgyfb8']
[2023-11-20T08:38:36.346+0000] {standard_task_runner.py:85} INFO - Job 6: Subtask retrieve_data
[2023-11-20T08:38:36.428+0000] {task_command.py:416} INFO - Running <TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-02T00:00:00+00:00 [running]> on host 74ca1234e792
[2023-11-20T08:38:36.511+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='austin_animal_shelter_dag' AIRFLOW_CTX_TASK_ID='retrieve_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-02T00:00:00+00:00'
[2023-11-20T08:38:36.518+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/Scripts/etl.py", line 18, in retrieve_data
    aws_access_key_id=YOUR_ACCESS_KEY,
NameError: name 'YOUR_ACCESS_KEY' is not defined
[2023-11-20T08:38:36.530+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=austin_animal_shelter_dag, task_id=retrieve_data, execution_date=20231102T000000, start_date=20231120T083836, end_date=20231120T083836
[2023-11-20T08:38:36.561+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 6 for task retrieve_data (name 'YOUR_ACCESS_KEY' is not defined; 75)
[2023-11-20T08:38:36.598+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-20T08:38:36.624+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-20T08:43:23.096+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-02T00:00:00+00:00 [queued]>
[2023-11-20T08:43:23.104+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-02T00:00:00+00:00 [queued]>
[2023-11-20T08:43:23.106+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-20T08:43:23.127+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): retrieve_data> on 2023-11-02 00:00:00+00:00
[2023-11-20T08:43:23.135+0000] {standard_task_runner.py:57} INFO - Started process 76 to run task
[2023-11-20T08:43:23.140+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'austin_animal_shelter_dag', 'retrieve_data', 'scheduled__2023-11-02T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpm18owevk']
[2023-11-20T08:43:23.150+0000] {standard_task_runner.py:85} INFO - Job 6: Subtask retrieve_data
[2023-11-20T08:43:23.247+0000] {task_command.py:416} INFO - Running <TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-02T00:00:00+00:00 [running]> on host c84b35a13244
[2023-11-20T08:43:23.316+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='austin_animal_shelter_dag' AIRFLOW_CTX_TASK_ID='retrieve_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-02T00:00:00+00:00'
[2023-11-20T08:43:25.928+0000] {etl.py:60} INFO - No new or updated data to upload. Creating an empty CSV with headers.
[2023-11-20T08:43:25.930+0000] {etl.py:64} INFO - Uploading data to 'raw_data'...
[2023-11-20T08:43:26.082+0000] {etl.py:71} INFO - Updating 'Old_Data' with current data...
[2023-11-20T08:43:27.213+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-20T08:43:27.295+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=austin_animal_shelter_dag, task_id=retrieve_data, execution_date=20231102T000000, start_date=20231120T084323, end_date=20231120T084327
[2023-11-20T08:43:27.429+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-20T08:43:27.487+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-11-20T08:46:36.621+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-02T00:00:00+00:00 [queued]>
[2023-11-20T08:46:36.637+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-02T00:00:00+00:00 [queued]>
[2023-11-20T08:46:36.642+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-20T08:46:36.668+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): retrieve_data> on 2023-11-02 00:00:00+00:00
[2023-11-20T08:46:36.683+0000] {standard_task_runner.py:57} INFO - Started process 90 to run task
[2023-11-20T08:46:36.700+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'austin_animal_shelter_dag', 'retrieve_data', 'scheduled__2023-11-02T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpcqqv6365']
[2023-11-20T08:46:36.707+0000] {standard_task_runner.py:85} INFO - Job 9: Subtask retrieve_data
[2023-11-20T08:46:36.820+0000] {task_command.py:416} INFO - Running <TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-02T00:00:00+00:00 [running]> on host d26cbef9e8ab
[2023-11-20T08:46:36.910+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='austin_animal_shelter_dag' AIRFLOW_CTX_TASK_ID='retrieve_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-02T00:00:00+00:00'
[2023-11-20T08:46:39.244+0000] {etl.py:62} INFO - No new or updated data to upload. Creating an empty CSV with headers.
[2023-11-20T08:46:39.246+0000] {etl.py:66} INFO - Uploading data to 'raw_data'...
[2023-11-20T08:46:39.418+0000] {etl.py:73} INFO - Updating 'Old_Data' with current data...
[2023-11-20T08:46:39.967+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-20T08:46:39.974+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=austin_animal_shelter_dag, task_id=retrieve_data, execution_date=20231102T000000, start_date=20231120T084636, end_date=20231120T084639
[2023-11-20T08:46:40.003+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-20T08:46:40.029+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-11-20T08:50:16.754+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-02T00:00:00+00:00 [queued]>
[2023-11-20T08:50:16.773+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-02T00:00:00+00:00 [queued]>
[2023-11-20T08:50:16.773+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-20T08:50:16.809+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): retrieve_data> on 2023-11-02 00:00:00+00:00
[2023-11-20T08:50:16.817+0000] {standard_task_runner.py:57} INFO - Started process 82 to run task
[2023-11-20T08:50:16.846+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'austin_animal_shelter_dag', 'retrieve_data', 'scheduled__2023-11-02T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmp6_x6i70c']
[2023-11-20T08:50:16.853+0000] {standard_task_runner.py:85} INFO - Job 10: Subtask retrieve_data
[2023-11-20T08:50:16.995+0000] {task_command.py:416} INFO - Running <TaskInstance: austin_animal_shelter_dag.retrieve_data scheduled__2023-11-02T00:00:00+00:00 [running]> on host 3ef39241d6e6
[2023-11-20T08:50:17.129+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='austin_animal_shelter_dag' AIRFLOW_CTX_TASK_ID='retrieve_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-02T00:00:00+00:00'
[2023-11-20T08:50:19.755+0000] {etl.py:87} INFO - No new or updated data to upload. Creating an empty CSV with headers.
[2023-11-20T08:50:19.758+0000] {etl.py:91} INFO - Uploading data to 'raw_data'...
[2023-11-20T08:50:19.897+0000] {etl.py:98} INFO - Updating 'Old_Data' with current data...
[2023-11-20T08:50:21.161+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-20T08:50:21.220+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=austin_animal_shelter_dag, task_id=retrieve_data, execution_date=20231102T000000, start_date=20231120T085016, end_date=20231120T085021
[2023-11-20T08:50:21.283+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-20T08:50:21.469+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
